pmg/templates/attendance_overview.html

```diff
@@ -4,101 +4,111 @@
 
 {% block page %}
 <section class="committee-attendance-overview">
-  <h1><i class="fa fa-fw fa-table" aria-hidden="true"></i> Committee meeting attendance trends for {{ year }}</h1>
-  <p>This page shows the attendance rates of the committees in the <a href="#national-assembly">National Assembly</a> and the <a href="#ncop">National Council of Provinces</a> in {{ year }} as well as the change in attendance since {{ year - 1 }}. The attendance rate is automatically updated as committees meet and attendance is recorded. In the first few weeks of a new year, when no committees have met yet, this page will show data of the previous year.</p>
+    <h1><i class="fa fa-fw fa-table" aria-hidden="true"></i> Committee meeting attendance trends for {{ year }}</h1>
+    
+  <p>This page shows the attendance rates of the committees in the <a href="#national-assembly">National Assembly</a> and the <a href="#ncop">National Council of Provinces</a> in {{year}}. The attendance rate is automatically updated as committees meet and attendance is recorded.</p>
   <p>You can click on a committee name to get more information; historical attendance rates as well as membership details can be found in the right sidebar of every committee page. It's also possible to download the raw data to do your own calculations and analysis.</p>
   <p>For MP and Minister attendance rates we'd like to refer you to <a href="https://www.pa.org.za/mp-attendance/">People's Assembly</a>.</p>
   <a class="btn btn-sm btn-success" href="https://api.pmg.org.za/committee-meeting-attendance/data.xlsx">Download data</a>
+  <a class="btn btn-sm btn-info" href="{{ url_for('archived_attendance_overview')}}">Historical Attendance</a>
 
   <h2 id="national-assembly">National Assembly</h2>
+  {% if not attendance %}
+  <h4>Data for current parliament is not avaliable yet.</h4>
+  {% else %}
   <table class="table table-hover sortable">
-    <thead>
-      <th class="th-rank"></th>
-      <th class="th-cte-name">Committee</th>
-      <th class="th-number-meetings hidden-xs" data-firstsort="desc">Meetings</th>
-      <th class="th-attendance" data-defaultsort="desc">Attendance</th>
-      <th class="th-attendance-change">Since {{ year - 1}}</th>
```

Comment:
This will be wrong in 6 months - we need to make this dynamic now or ensure we fix it statically or dynamically so it doesn't show the wrong thing in January

Reply:
updated this to reflect what's needed https://www.pivotaltracker.com/story/show/166897271